README.rst

```diff
@@ -128,13 +128,21 @@ own rotozoom function.  The surfarray module requires the Python
 NumPy package for its multidimensional numeric arrays.
 Dependency versions:
 
-* CPython >= 3.6 or PyPy3
-* SDL >= 2.0.4
-* SDL_mixer >= 2.0.0
-* SDL_image >= 2.0.2
-* SDL_ttf >= 2.0.11
-* SDL_gfx (optional, vendored in)
-* NumPy >= 1.6.2 (optional)
++----------+------------------------+
+| CPython  | >= 3.6 (Or use PyPy3)  |
++----------+------------------------+
+| SDL      | >= 2.0.4               |
++----------+------------------------+
+| SDL_mixer| >= 2.0.0               |
++----------+------------------------+
+| SDL_image| >= 2.0.2               |
++----------+------------------------+
+| SDL_ttf  | >= 2.0.11              |
++----------+------------------------+
+| SDL_gfx  | (Optional, vendored in)|
++----------+------------------------+
+| NumPy    | >= 2.6.2 (Optional)    |
```

Comment:
You have changed the version number here for NumPy.

Reply:
@MyreMylar I have fixed this.